src/interface/whitelist/index.js

```diff
@@ -0,0 +1,17 @@
+const express = require('express');
+
+const router = express.Router();
+
+const {
+    asyncHandler,
+    asyncHandlerArray,
+} = require('../../infra/asyncHandler');
+
+const whitelistAddress = require('./address');
+const getAddress = require('./getAddress');
+const { whitelistAdmin } = require('../middleware');
+
+router.post('/address', asyncHandler(whitelistAdmin), asyncHandlerArray(whitelistAddress));
```

Comment:
you can even make it shorter and use this: `router.post('/', asyncHandler(whitelistAdmin), asyncHandlerArray(whitelistAddress));`

Reply:
Keeping the URL` '/address'...`  of this format, because later we'll have` /codes` endpoint too

Reply:
base route IMO should be `/whitelist` 
then sub routes could be
/whitelist/code
/whitelist/check/:address
/whitelist/address/update

Reply:
check will be required for code as well. 
Hence keeping 
/whitelist/address : for address related queries
/whitelist/codes: for codes related queries